src/inproc.rs

```diff
@@ -60,11 +47,14 @@ pub fn unregister_keys(registry_keys_to_remove: Vec<RegistryKeyInfo>) -> HRESULT
     hr
 }
 
-fn create_class_key(key_info: &RegistryKeyInfo) -> Result<HKEY, LSTATUS> {
-    let hk_result: HKEY = std::ptr::null_mut::<c_void>() as HKEY;
-    let lp_class = std::ptr::null_mut::<CHAR>();
-    let lp_security_attributes = std::ptr::null_mut::<SECURITY_ATTRIBUTES>();
-    let lpdw_disposition = std::ptr::null_mut::<DWORD>();
+const HKEY_CLASSES_ROOT: *mut c_void = 0x8000_0000 as *mut c_void;
```

Comment:
Pretty sure this should be u32

Reply:
`typedef DWORD HKEY` in the header

Reply:
The examples no longer work when I make this change.

Reply:
Compile or runtime? Weird